gitgud/__main__.py

```diff
@@ -224,26 +223,25 @@ def handle_challenges(self, args):
             print("Challenges for level \"{}\" : \n".format(level.name))
 
         
-        for index, challenge in enumerate(level.challenges):
+        for index, challenge in enumerate(level):
             print(str(index + 1) + ": " + challenge.name)
 
     def handle_load(self, args):
         self.assert_initialized()
-        if args.level_name in all_levels:
+        if args.level_name in all_levels.keys():
             level = all_levels[args.level_name]
 
             if args.challenge_name is not None:
-                if args.challenge_name in all_levels[args.level_name].challenges:
-                    challenge = level.challenges[args.challenge_name]
+                if args.challenge_name in all_levels[args.level_name].keys():
```

Comment:
Same as all_levels, you should be able to use what you had before. If not, there is a `__contains__` method you can implement